tracer/src/Datadog.Trace/IAST/IastModule.cs

```diff
@@ -6,6 +6,7 @@
 #nullable enable
```

Comment:
Iast is a class analog to Security.cs and, besides holding settings, it adds the Iast specific instrumentation. If, in the future, we want to do things like reloading IAST config or remove some instrumentation, this is the class that would do it. On the other hand, IastModule holds the logic of what needs to be done when hitting a instrumented spot and basically, contains static methods. It would probably be interesting declaring IastModule as static class. Accessing to Iast.Instance.Settings would remove the need to pass iast as parameter or maybe it would be cleaner to send only the settings. I will take a look. Anyway, all this, tough interesting, would be out of the scope of this PR.